src/NClient.Providers/NClient.Providers.Api.Rest/RestRequestBuilder.cs

```diff
@@ -115,14 +117,46 @@ internal class RestRequestBuilder : IRequestBuilder
                 throw _clientValidationExceptionFactory.MultipleBodyParametersNotSupported();
             if (bodyParams.Length == 1)
```

Comment:
Can we revert if and reduce nested levels?

Reply:
Done